modules/local/zippy/main.nf

```diff
@@ -0,0 +1,21 @@
+process ZIP_OUTPUT {
+    tag 'Zip output files'
+    label "process_single"
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/p7zip:16.02' :
+        'biocontainers/p7zip:16.02' }"
+    input:
+    path arborview_files
+    path arborator_files
+    path software_version_file
+
+    output:
+    path "*.zip"
+
+    script:
+    """
+    # -l : Don't store symlinks, store the files they point to
+    7za a -l "arboratornf.zip" $arborview_files $arborator_files $software_version_file
```

Comment:
When unzipping `arboratornf.zip`, I noticed it unzips into the current directory. Could you change this to add all files into a single directory `arboratornf/`. This directory should contain as subdirectories/subfiles:

1. `arboratornf/arborator`: The arborator results
2. `arboratornf/arborview`: The arborview results.
3. `arboratornf/software_versions.yml`

Reply:
Based on discussions with @jrober84, it's okay to have all files in the same directory.

Also, looks like unzipping in current directory only happens if you use `unzip` command-line tool. Using even the Linux GUI to unzip, they all get written into an `arboratornf/` directory. So this change is not needed.